src/client/clouds.cpp

```diff
@@ -223,94 +231,131 @@ void Clouds::updateMesh()
 		const f32 ry = m_enable_3d ? m_params.thickness * BS : 0.0f;
 		const f32 rz = cloud_size / 2;
 
-		for(u32 i = 0; i < num_faces_to_draw; i++)
+		bool soft_clouds_enabled = g_settings->getBool("soft_clouds");
+		for (u32 i = 0; i < num_faces_to_draw; i++)
 		{
-			switch(i)
+			switch (i)
 			{
 			case 0:	// top
-				for (video::S3DVertex &vertex : v) {
-					vertex.Normal.set(0,1,0);
+				for (video::S3DVertex& vertex : v) {
+					vertex.Normal.set(0, 1, 0);
 				}
-				v[0].Pos.set(-rx, ry,-rz);
+				v[0].Pos.set(-rx, ry, -rz);
 				v[1].Pos.set(-rx, ry, rz);
-				v[2].Pos.set( rx, ry, rz);
-				v[3].Pos.set( rx, ry,-rz);
+				v[2].Pos.set(rx, ry, rz);
+				v[3].Pos.set(rx, ry, -rz);
```

Comment:
I'm seeing now, I've kind of messed up the whitespaces and stuff, I should probably fix that later